src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,113 @@
+package core.basesyntax.service;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    public static final int MINIMUM_ALLOWABLE_AGE = 18;
+    private static final String MINIMUM_ALLOWABLE_QNTY_CHARACTERS_PASSWORD = "qwerty";
```

Comment:
```suggestion
    private static final int MINIMUM_ALLOWABLE_QNTY_CHARACTERS_PASSWORD = 6;
```

I think it's better just to specify min number of characters

Reply:
changed